### PR TITLE
hostctl: Add version 0.5.3

### PR DESCRIPTION
**Pull request by guumaster**
This CLI tool allows you to manage your hosts file by profiles with ease. Check [hostctl home](https://github.com/guumaster/hostctl)

### CODE REVIEW

File: bucket/hostctl.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "url": "https://github.com/guumaster/hostctl/releases/download/v0.5.3/hostctl_0.5.3_windows_64-bit.zip",
+            "bin": [
+                "hostctl.exe"
+            ],
+            "hash": "31d9b0b262f2079661164113e6c59a2ca2d3cb4998b4e9466c3b55a0a89be6ec"
+        }
+    },
+    "homepage": "https://github.com/guumaster/hostctl",
+    "license": "MIT",
+    "description": "manage your hosts file like a pro",
```
Comment (Ash258) [inline bucket/hostctl.json]:
```suggestion
{
    "version": "0.5.3",
    "description": "Hosts file manager",
    "homepage": "https://github.com/guumaster/hostctl",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://github.com/guumaster/hostctl/releases/download/v0.5.3/hostctl_0.5.3_windows_64-bit.zip",
            "hash": "31d9b0b262f2079661164113e6c59a2ca2d3cb4998b4e9466c3b55a0a89be6ec"
        }
    },
    "bin": "hostctl.exe",
```

File: bucket/hostctl.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/guumaster/hostctl/releases/download/$version/hostctl_$version_windows_64-bit.zip"
```
Comment (Ash258) [inline bucket/hostctl.json]:
```suggestion
                "url": "https://github.com/guumaster/hostctl/releases/download/v$version/hostctl_$version_windows_64-bit.zip"
```

**Comment by guumaster** [comment]
Should I do something else for this PR? or is on the queue to be merged?

**Comment by guumaster** [comment]
thanks!

**Pull request merged**